package.json

```diff
@@ -364,6 +364,11 @@
                         ],
                         "description": "Endpoint of the user account."
                     },
+                    "leetcode.useEndpointTranslation": {
+                        "type": "boolean",
+                        "default": true,
+                        "description": "Use endpoint's translation (if available)"
```

Comment:
Please explicitly set the scope for the setting. I think it can be `application` here.

Reply:
Thank you for your comment! This is fixed.